packages/components/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@kaizen/components",
+  "version": "1.0.0",
+  "description": "Kaizen component library",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "future",
+    "src",
+    "dist"
+  ],
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf 'dist'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git"
+  },
+  "author": "Geoffrey Chong <geoff.chong@cultureamp.com>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/cultureamp/kaizen-discourse/issues"
+  },
+  "homepage": "https://github.com/cultureamp/kaizen-design-system#readme",
+  "devDependencies": {
+    "rimraf": "^3.0.2"
```

Comment:
Side thought: I wonder if it's possible to just have this in the root of the repo? 🤔

Reply:
Oh, maybe 🤔

Reply:
Yeah that works, removing the dep here.